cmd/gitbase/command/server.go

```diff
@@ -344,12 +362,20 @@ func (c *Server) addDirectory(d directory) error {
 	return nil
 }
 
+type bareness int
```

Comment:
hmm, maybe just `bareOpt`

Reply:
I found `bareness` funny but `bareOpt` may be clearer. :+1:

Reply:
Changed